src/linear_algebra/quotient.lean

```diff
@@ -346,6 +346,21 @@ begin
   exact inf_le_right,
 end
 
+/-- If `P` is a submodule of `M` and `Q` a submodule of `N`,
+and `f : M ≃ₗ N` maps `P` to `Q`, then `M ⧸ P` is equivalent to `N ⧸ Q`. -/
+@[simps] def quotient.equiv {N : Type*} [add_comm_group N] [module R N]
```

Comment:
Can you add the lemma that says when f is equiv.refl, the resulting equiv is linear_equiv.of_eq? Similarly a lemma for `f.symm` would be handy.